adapters/dmx/dmx.go

```diff
@@ -263,8 +263,7 @@ func (adapter *DmxAdapter) MakeBids(request *openrtb2.BidRequest, externalReques
 }
 
 func fetchParams(params dmxExt, inst openrtb2.Imp, ins openrtb2.Imp, imps []openrtb2.Imp, banner *openrtb2.Banner, video *openrtb2.Video, intVal int8) []openrtb2.Imp {
-	var tempimp openrtb2.Imp
-	tempimp = inst
+	var tempimp openrtb2.Imp = inst
```

Comment:
We should prefer `tempimp := inst` here.

Reply:
change in a36d18ca00fe56181dc2b01e0a4bcaadcac0f5d5 as well